app/doc/responses.go

```diff
@@ -121,3 +121,44 @@ type groupsGetGroupProgressResponse struct {
 		AvgTimeSpent float32 `json:"avg_time_spent"`
 	}
 }
+
+// OK. Success response with teams progress on items
+// swagger:response groupsGetTeamProgressResponse
+type groupsGetTeamProgressResponse struct {
+	// in: body
+	Body []struct {
+		// The team’s `group_id`
+		// required:true
+		GroupID int64 `json:"group_id,string"`
+		// required:true
+		ItemID int64 `json:"item_id,string"`
+		// The best score across all attempts, or 0
```

Comment:
Was more a way to explain to the implementer.
We can say "Current score.  If there are no attempts, the score is 0."

Reply:
fixed